NickvisionTubeConverter.GNOME/Program.cs

```diff
@@ -108,4 +108,15 @@ private async void OnActivate(Gio.Application sender, EventArgs e)
             await _mainWindow.StartAsync();
         }
     }
+
+    /// <summary>
+    /// Occurs when the application is opened
+    /// </summary>
+    /// <param name="sender">Gio.Application</param>
+    /// <param name="e">Gio.Application.OpenSignalArgs</param>
+    private void OnOpen(Gio.Application sender, Gio.Application.OpenSignalArgs e)
+    {
+        _mainWindowController.UrlToLaunch = e.Files[0].GetUri();
+        _application.Activate();
```

Comment:
this will call `_mainWindow!.Present()` but will not do anything with UriToLaunch, no?

Reply:
The activates the application, which triggers the OnActivate event handler above this method which starts the MainWindow

Reply:
RunWithSynchronizationContext doesn't accept args though, no?

Only in 0.5.0 preview it does.

Reply:
```csharp
if (_mainWindow != null)
        {
            _mainWindow!.SetVisible(true);
            _mainWindow!.Present();
        }
```

Reply:
Ah you're right, my bad let me fix that

Reply:
All fixed in the latest commit.